starboard/evergreen/shared/platform_configuration/BUILD.gn

```diff
@@ -34,51 +34,10 @@ config("platform_configuration") {
   if (sb_is_evergreen) {
     ldflags += [ "-nostdlib" ]
   }
-  cflags = [
-    "-ffunction-sections",
-    "-fdata-sections",
-    "-fPIC",
-    "-nostdlibinc",
-    "-isystem" + rebase_path("//third_party/llvm-project/libcxxabi/include",
-                             root_build_dir),
-    "-isystem" + rebase_path("//third_party/llvm-project/libunwind/include",
-                             root_build_dir),
-    "-isystem" + rebase_path("//third_party/llvm-project/libcxx/include",
-                             root_build_dir),
-    "-isystem" + rebase_path("//third_party/musl/include", root_build_dir),
-    "-isystem" + rebase_path("//third_party/musl/arch/generic", root_build_dir),
-  ]
 
-  cflags_cc = [
-    "-nostdinc++",
-    "-std=c++17",
-  ]
+  configs = [ "//starboard/build/config/modular/shared:modular_config" ]
```

Comment:
Define configs are the top of this config

Reply:
done